brave/brave-core
# Modify reset sync

Fix https://github.com/brave/brave-browser/issues/7412

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues/7412) for my issue if one did not already exist.
- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
Use steps from https://github.com/brave/brave-browser/issues/7412 

1. Create a sync chain
2. Delete the chain
3. Try re-creating it
4. Add another device to this chain and ensure sync works

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
